packages/docusaurus-theme-classic/src/theme/Tabs/index.js

```diff
@@ -31,7 +31,11 @@ function Tabs(props) {
         ))}
       </ul>
       <div className="margin-vert--md">
-        {[...children].filter(child => child.props.value === selectedValue)[0]}
+        {
```

Comment:
How about we use `React.Children` instead? Better to stick to what React provides

Reply:
oh yeah, fixed (although `Array.from` is a bulletproof solution), probably it was necessary to use it initially, but I prevented it :(